project/AkkaBuild.scala

```diff
@@ -216,6 +215,74 @@ object AkkaBuild extends Build {
     dependencies = Seq(persistence % "test->test", testkit % "test->test", remote % "test", protobuf)
   )
 
+  lazy val httpCore = Project(
+    id = "akka-http-core",
+    base = file("akka-http-core"),
+    dependencies = Seq(stream, parsing, streamTestkit % "test->test")
+  )
+
+  lazy val http = Project(
+    id = "akka-http-experimental",
+    base = file("akka-http"),
+    dependencies = Seq(httpCore)
+  )
+
+  lazy val httpTestkit = Project(
+    id = "akka-http-testkit",
+    base = file("akka-http-testkit"),
+    dependencies = Seq(http, streamTestkit)
+  )
+
+  lazy val httpTests = Project(
+    id = "akka-http-tests",
+    base = file("akka-http-tests"),
+    dependencies = Seq(
+      httpTestkit % "test", streamTestkit % "test->test", testkit % "test->test", httpSprayJson, httpXml, httpJackson,
+      multiNodeTestkit, remoteTests % "test->test") // required for multi-node latency/throughput Spec
+  ).configs(MultiJvm)
+
+  lazy val httpMarshallersScala = Project(
+    id = "akka-http-marshallers-scala-experimental",
+    base = file("akka-http-marshallers-scala")
+  )
+  .settings(parentSettings: _*)
+  .aggregate(httpSprayJson, httpXml)
+
+  lazy val httpXml =
+    httpMarshallersScalaSubproject("xml")
+
+  lazy val httpSprayJson =
+    httpMarshallersScalaSubproject("spray-json")
+
+  lazy val httpMarshallersJava = Project(
+    id = "akka-http-marshallers-java-experimental",
+    base = file("akka-http-marshallers-java")
+  )
+  .settings(parentSettings: _*)
+  .aggregate(httpJackson)
+
+  lazy val httpJackson =
+    httpMarshallersJavaSubproject("jackson")
+
+  def httpMarshallersScalaSubproject(name: String) =
+    Project(
+      id = s"akka-http-$name-experimental",
+      base = file(s"akka-http-marshallers-scala/akka-http-$name"),
+      dependencies = Seq(http)
+    )
+
+  def httpMarshallersJavaSubproject(name: String) =
+    Project(
+      id = s"akka-http-$name-experimental",
+      base = file(s"akka-http-marshallers-java/akka-http-$name"),
+      dependencies = Seq(http)
+    )
+
+  lazy val parsing = Project(
+    id = "akka-parsing",
+    base = file("akka-parsing")
+  )
+
```

Comment:
Looks something went wrong when I tried to fix the merge conflicts ... thanks for noticing. I'll fix it again and update the PR.